src/components/controlled/controlledInput/ControlledInput.tsx

```diff
@@ -1,13 +1,17 @@
+import { JSX } from 'react'
+
 import { FieldValues, useController, UseControllerProps } from 'react-hook-form'
 
 import { Input, InputProps } from '@/components'
 
 type ControlledInputProps<T extends FieldValues> = UseControllerProps<T> &
   Omit<InputProps, 'value' | 'onChange'>
 
-export const ControlledInput = <T extends FieldValues>(props: ControlledInputProps<T>) => {
-  const { name, control, ...restInputProps } = props
-
+export const ControlledInput = <T extends FieldValues>({
+  name,
+  control,
+  ...restInputProps
```

Comment:
maybe better use naming ...restProps. Need to compare with ui components and do one similar variant

Reply:
Done!